app/controllers/moves_type.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
-require_relative 'human_move'
-require_relative 'computer_move'
+require_relative "human_move"
+require_relative "computer_move"
 
 class MovesType
   def get_move_type(current_player)
-    if current_player == 'human'
+    if current_player == "human"
```

Comment:
I see we have these strings `human` and `computer` throughout the codebase. Can we create constants for them so that they are reusable?